oak_containers_launcher/src/server.rs

```diff
@@ -160,7 +162,16 @@ impl Launcher for LauncherServerImplementation {
         &self,
         request: Request<SendAttestationEvidenceRequest>,
     ) -> Result<Response<()>, tonic::Status> {
+        let request = request.into_inner();
+        let evidence = request.dice_evidence.ok_or_else(|| {
+            tonic::Status::internal("send_attestation_evidence_request doesn't have evidence")
+        })?;
+        // TODO(#4627): Remove old `AttestationEvidence` message.
         #[allow(deprecated)]
+        let deprecated_evidence = request.evidence.ok_or_else(|| {
+            tonic::Status::internal("send_attestation_evidence_request doesn't have evidence")
```

Comment:
this message should probably be a bit different from the one above to make it easier to find the source of an error.